src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Unix.cs

```diff
@@ -160,6 +161,10 @@ protected override void Dispose(bool disposing)
 
         protected internal override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
+            if (DiagnosticsHandler.IsEnabled())
```

Comment:
> DiagnosticSource does not have a no-argument IsEnabled() API.

I see.  I just assumed it had a similar scheme as EventSource, with an IsEnabled() that was broad and then an overload that was more specific to a particular set of events.